site/en/guides/border-router/build.md

```diff
@@ -40,16 +40,28 @@ Clone the OTBR repository:
 $ git clone https://github.com/openthread/ot-br-posix
 ```
 
-Install dependencies:
+Install dependencies. For a complete list of OTBR default
```

Comment:
It might be worth making level 3 headers for Install Dependencies and Compile/Install OTBR since with the extra commands it all runs together now.  Or have a level 3 for Default Usage and Beaglebone Black where the commands can be clearly grouped.  WDYT?

Reply:
Sure, I think something like that would look good...I'll make some updates...